### ISSUE TITLE
Array bug

### ISSUE THREAD

**Issue by MadRoxx** [Zeex/pawn#67]
After i upgrade pawn to new version, my arrays dont works correctly. 
Example:
```
[20:50:06] -mSelection- WARNING: Failed to load list: "r/furdo.txt"
[20:50:06] -mSelection- WARNING: Failed to load list: "rdo.txt"
[20:50:06] -mSelection- WARNING: Failed to load list: "txt"
[20:50:06] -mSelection- WARNING: Failed to load list: ""
[20:50:06] -mSelection- WARNING: Failed to load list: ""
[20:50:06] -mSelection- WARNING: Failed to load list: "Ágyak"
[20:50:06] -mSelection- WARNING: Failed to load list: "k"
[20:50:06] -mSelection- WARNING: Failed to load list: ""
```
My array:
```
#define BUTORKATEGORIAK 9
new g_ButorKategoriak[BUTORKATEGORIAK][2][24] =
{
	{ "Fürdőszoba", "butor/furdo.txt" },
	{ "Ágyak", "butor/agyak.txt" },
	{ "Székek", "butor/szekek.txt" },
	{ "Asztalok", "butor/asztalok.txt" },
	{ "Ajtók", "butor/ajtok.txt" },
	{ "Műszaki cikkek", "butor/muszaki.txt" },
	{ "Lámpák", "butor/lampak.txt" },
	{ "Konyha", "butor/konyha.txt" },
	{ "Fitnessz", "butor/fitnessz.txt" }
};
new g_ButorMenu[BUTORKATEGORIAK] = {mS_INVALID_LISTID, ...};
```

After i downgrade pawno, all works fine again.

**Comment by ziggi** [comment]
Reported here: #63
Fixed here: 1cca8af0d2c6d290c095f27f39bd941e4374054e

**Comment by MadRoxx** [comment]
Oops, then sorry. :D Thank you!
